### PR TITLE
[IMP] adapt snippet design

### PR DESCRIPTION
**Pull request by chgo-odoo**
- Requires :
https://github.com/odoo/odoo/pull/172728
https://github.com/odoo/enterprise/pull/66578

task-3670312

### CODE REVIEW

**Review by stefanorigano — APPROVED** [review]
Great, LGTM, thanks @chgo-odoo 👍
.. waiting for linked PRs...

**Comment by sobo-odoo** [comment]
@robodoo delegate=bvr-odoo

File: theme_vehicle/views/customizations.xml [hunk 1]
```diff
@@ -115,7 +115,7 @@
 <template id="s_call_to_action" inherit_id="website.s_call_to_action" name="Vehicle s_call_to_action">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_cc5 pt96 pb64" remove="o_cc3 pt48 pb24" separator=" "/>
+        <attribute name="class" add="o_cc5 pt96" remove="o_cc5 pt64" separator=" "/>
```
Comment (bvr-odoo) [inline theme_vehicle/views/customizations.xml:118]: no need to remove then add `o_cc5` :wink:

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
A small change needed, otherwise all good. Thanks.

**Comment by chgo-odoo** [comment]
Thanks @bvr-odoo, changes done, should be good, normally, could you have a look, please?

**Pull request closed without merging**